### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
Add a CONTRIBUTING.md for the organization. This was taken from the mirror node repo and updated to be non-project specific, adjust support channels to prioritize GH discussions and generally updated.

**List of changes**:
- Add a CONTRIBUTING.md for the organization

**Related issue(s)**:

**Notes for reviewer**:

**Checklist**

- [x] Documentation added
- [ ] Tests added

### CODE REVIEW

**Review by SimiHunjan — APPROVED** [review]
👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  area.
+- Clean up the title to succinctly and clearly state the issue (if needed).
+- Ask the submitter to clarify any items.
+- We attempt to do this process at least once per work day.
+
+#### Discussion
+
+- Issues should be connected to the [pull request](#pull-requests) that resolves it.
+- Whoever is working on an issue (whether a maintainer or someone from the community), should either assign the issue to
+  them self or add a comment that they will be working on it.
```
Comment (Nana-EC) [inline CONTRIBUTING.md]: nit: 
```suggestion
  themself or add a comment that they will be working on it.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  a [keyword](https://help.github.com/en/articles/closing-issues-using-keywords)
+  to automatically close the related issue.
+- Commits should be as small as possible, while ensuring that each commit is correct independently
+  (i.e., each commit should compile and pass tests).
+- Add tests and documentation relevant to the fixed bug or new feature. Code coverage should stay the same or increase
+  for the PR to be approved.
+- We more than welcome PRs that are currently in progress. If a PR is a work in progress, it should be opened as
+  a [Draft PR](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests). Once the PR is ready for
+  review, mark it as ready to convert it to a regular PR.
+- After submitting, ensure all GitHub checks pass before requesting a review. Also double check all static analysis and
```
Comment (Nana-EC) [inline CONTRIBUTING.md]:
```suggestion
- After submitting, ensure all GitHub checks pass before requesting a review. Also double-check all static analysis and
```

**Review by Nana-EC — COMMENTED** [review]
Looks good, just 2 nits

File: CONTRIBUTING.md (hunk 2 above)
Comment (steven-sheehy) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 1 above)
Comment (steven-sheehy) [reply CONTRIBUTING.md]: Updated

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**